openmc/material.py

```diff
@@ -64,6 +64,12 @@ class Material(object):
         List in which each item is a 3-tuple consisting of an
         :class:`openmc.Nuclide` instance, the percent density, and the percent
         type ('ao' or 'wo').
+    molar_mass : float
+        The molar mass of the material computed in units of grams per mole of
+        nuclides in a material. This entails that the molar mass does not depend
+        on the magnitude of the sum of atomic amounts of elements and nuclides
+        in the material. For instance, the molar mass of UO2 would be
+        ~90 g/mol.
```

Comment:
I agree - `average_molar_mass` seems more fitting. I changed the description to match yours as well.